.github/CONTRIBUTING.md

```diff
@@ -55,6 +55,7 @@ Run tests from the library:
 ```bash
 # using make
 make test
+make alpine-test
```

Comment:
I started checking this deeper, and this is the first thing that I found so far hehe

```suggestion
make test-alpine
```